Feb 2021 Newsletter (#105)
readthedocs/ethicalads.io: content/posts/newsletter-february-2021.md

tags: newsletter, community
authors: David Fischer
image: /images/posts/2021-feb-newsletter.jpg
image_credit: <span>Photo by <a href="https://unsplash.com/@szmigieldesign?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">Lukasz Szmigiel</a> on <a href="https://unsplash.com/?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">Unsplash</a></span>

[comment]: # (The queries run to get this data, then computed in a spreadsheet for publisher rev)
[comment]: # (?start_date=2020-08-01&end_date=2020-08-31&campaign_type=All+types&revenue_share_percentage=50.0)
[comment]: # (?start_date=2020-09-01&end_date=2020-09-30&campaign_type=paid&revenue_share_percentage=70.0&sort=revenue)

## February advertising stats

February was our best month ever for advertising.
Despite being a shorter month, we were still able to beat our previous bests from [January]({filename}/posts/newsletter-january-2021.md).

In February 2021:

* We generated **$11,671.47** for our publishers, a 5.5% increase over January.
* We had **106 publishers** on our network with a paid ad impression.
* We had **6,435,571** paid ad views across the world.

> * Read the Docs, our own publisher, generated **$17,404.14** on 7,165,169 paid ad impressions.

Reviewer: Odd we made so much more on a smaller number of impressions. Is this because we're realizing 100% of the revenue here?

Author: Yes

Reviewer: We should probably count this as 70% so that it doesn't look like we're biasing towards RTD.

> It's always hard to beat a previous record but we're expecting another positive month in March.

Reviewer: ```suggestion
We're working hard to grow revenue for all our publishers, and we're expecting another record month in March.
 ```

Author: I just wanted to hedge slightly in case we don't succeed but I guess go for bust!

Reviewer: Why not :D

If this role sounds interesting to you,
please [get in touch with any questions](mailto:ads@ethicalads.io?subject=Ads%20account%20manager%20role)
or [apply]({filename}/pages/jobs/account-manager-part-time.md)!
In addition, Read the Docs is hiring a [frontend developer](https://blog.readthedocs.com/job-frontend/).

Infrastructure will continue to cause some headaches in the next few months.
Read the Docs Community completed a move from Azure to AWS in February
and EthicalAds will probably follow suit in the next few months.
In addition, we've been hitting I/O limits as I mentioned above and while we've made improvements there
and our latency is pretty darn good, we still want to do more.
Our data points pretty directly to faster ads performing better.

## Upcoming features

The major features in our upcoming roadmap:

* We want to give both advertisers and publishers high level data right in their dashboard when they first login.
  These are stats like exactly how much advertising money they've spent or earned this month
  and how this month compares with last month.
  This feature is still on our roadmap but we didn't wrap it up in February.

> * We're finally going to expand our advertising logic to take into account price when selecting ads.

Reviewer: ```suggestion
* We're going to expand our advertising logic to take into account price when selecting ads.
```

>   At least one advertiser has agreed to be a flex advertiser with part of their ad buys in Q2

Reviewer: ```suggestion
  One advertiser has agreed to be a flex advertiser with part of their ad buys in Q2
```
